odoo.conf

```diff
@@ -0,0 +1,87 @@
+[options]
+##
+#Addons Path
+##
+addons_path = C:\GitHub\OpenERP\Odoo 13\addons, C:\BitBucket\JetPERU\ERP 13\core_13, C:\BitBucket\JetPERU\ERP 13\web_13
+bin_path = False
+##
+# Urls Logs
+##
+logfile = False
+# logfile = C:\GitHub\OpenERP\Odoo 13\odoo.log
+log_db = False
+log_db_level = warning
+log_handler = :INFO
+log_level = info
+logrotate = False
+syslog = False
+##
+# DataBase Configure
+##
+pg_path = C:\Program Files\PostgreSQL\10\bin
+db_host = localhost
+db_maxconn = 64
+db_name = False
+db_password = openpgpwd
+db_port = 5432
+db_sslmode = prefer
+db_template = template1
+db_user = openpg13
+dbfilter = 
+# dbfilter = JETERP13
+list_db = True
+# list_db = False
+##
+# Test Configure
+##
+test_commit = False
+test_enable = False
+test_file = False
+test_report_directory = False
+##
+# Control CPU Configure
+##
+limit_memory_hard = None
+limit_memory_soft = None
+limit_request = None
+limit_time_cpu = 60
+limit_time_real = 120
+limit_time_real_cron = None
+max_cron_threads = 2
+osv_memory_age_limit = 1.0
+osv_memory_count_limit = False
+workers = 25
+##
+# Mail Configure
```

Comment:
SMTP section really matters?
We now use it all via outgoing sever IMHO.
Curious.

Reply:
hi :D, by my mistake of not having configured my IDE correctly, I ended up doing pull request to repo.